crates/turbo-tasks/src/macro_helpers.rs

```diff
@@ -0,0 +1,21 @@
+//! Runtime helpers for [turbo-tasks-macro].
+pub use once_cell::sync::{Lazy, OnceCell};
+pub use tracing;
+
+pub use super::manager::{find_cell_by_type, notify_scheduled_tasks, spawn_detached};
+use crate::debug::ValueDebugFormatString;
+
+#[inline(never)]
+pub async fn value_debug_format_field(value: ValueDebugFormatString<'_>) -> String {
```

Comment:
I modified this to accept `ValueDebugFormatString` instead of `dyn (ValueDebugFormat + Sync)` because we need deref coercion to make macro work